java/src/jmri/jmrit/roster/swing/CopyRosterGroupAction.java

```diff
@@ -86,18 +86,19 @@ public void actionPerformed(ActionEvent event) {
         }
 
         String entry = (String) JOptionPane.showInputDialog(_who,
-                "<html><b>Duplicate roster group</b><br>Enter the name for the new roster group.</html>",
-                "Duplicate Roster Group " + group,
+                Bundle.getMessage("DuplicateRosterGroupNewName"),
+                Bundle.getMessage("DuplicateRosterGroupTitle",group),
                 JOptionPane.INFORMATION_MESSAGE,
                 null,
                 null,
                 null);
-        if (entry == null || entry.equals(Roster.ALLENTRIES)) {
+        entry = entry.trim(); // remove white space around name, also prevent "Space" as a Group name
```

Comment:
Do you know that roster groups are elsewhere prevented from having leading or trailing spaces? I don't think they are, so this may break existing roster groups for users.